com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+## [1.0.1] - 
```

Comment:
Is this the release tag we want, or are we still putting bugfixes under the 1.0.0 section?